public-src/phphants/scss/admin.scss

```diff
@@ -47,6 +47,17 @@ a.link-button {
     padding: 7px 20px;
     font-size: 14px;
     text-decoration: none;
+    @media (max-width: 320px) {
+        display: inline-block;
+        margin: 5px auto 0 auto;
+        width: 100%;
+    }
+}
+.hide-mobile {
+    @media (max-width: 320px) {
+        display: none;
+        visibility: hidden;
```

Comment:
Just wondering, why use both `display` and `visibility` properties?

Reply:
Not sure, I'll take this out ;)